repositories/dbmodels/db_scenario_testrun.go

```diff
@@ -0,0 +1,29 @@
+package dbmodels
+
+import (
+	"time"
+
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/utils"
+)
+
+type DBScenarioTestRun struct {
+	Id                  string        `db:"id"`
+	ScenarioIterationId string        `db:"scenario_iteration_id"`
+	CreatedAt           time.Time     `db:"created_at"`
+	Period              time.Duration `db:"period"`
+	Status              string        `db:"status"`
+}
```

Comment:
I think we need to save the couple of iteration ids used on each backtest (name TBD):
- the `liveVersionID`
- the `phantomVersionID`

This is used for at least to good reasons :
1. Easilly display the couple of versions when we list past testruns
2. Handle special cases where a testrun is "up" but the live version has changed (still to define, but IMO we should "down" the testrun when the associated live version id deactivated)

Bonnus: I would also store a "reason" to explain why the testrun goes from `up` -> `down` (if we do automatically close some testruns, it could help users understand the internal logic). But it's not mandatory in a V0.

Reply:
Though, on second thought: I think I'm against storing the live version id on the test run, as this is nasty data denormalization, and if anything bad happens it's bound to create a form of data corruption (not the absolutely worst type of it, since this is "analytics", but still).
We can discuss this one tomorrow.

Reply:
(nb: or we store the "reference iteration id" - which is almost the same thing 99% of the time, but has another interpretation)